dspace-api/src/main/java/org/dspace/core/ProvenanceMessageTemplates.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.core;
+
+/**
+ * The ProvenanceMessageTemplates enum provides message templates for provenance messages.
+ *
+ * @author Michaela Paurikova (dspace at dataquest.sk)
+ */
+public enum ProvenanceMessageTemplates {
```

Comment:
is this class really required?
is it used without the MessageFormatter?

Reply:
It is not required and it is used only with ProvenanceService.
However, I suggest keeping it separate for better readability. Additionally, the ProvenanceService already has around 300 lines of code, so I would prefer not to add another constants there.